functions/src/modules/pm/groups/deleteGroup.ts

```diff
@@ -1,5 +1,28 @@
-import { ExpressFunction } from "../../../@types";
+import type { ExpressFunction } from "../../../@types";
+import { firebaseApp } from "../../../config/firebaseConfig";
+import {
+  expressErrorHandlerFactory} from "../../../helpers";
+import { uasPermissionSwitch } from "../../../systems/uas";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const deleteGroup: ExpressFunction = (req, res, next) => {
+    uasPermissionSwitch({
+        organizer: { accepted: execute },
+      })(req, res, next);
+};
 
-export default replaceMe;
+const execute: ExpressFunction = (req, res, next) => {
+    const errorHandler = expressErrorHandlerFactory(req, res, next);
+  
+    firebaseApp
+    .firestore()
+    .collection("PrizeCategoryGroup")
```

Comment:
Can we call this `prizeGroups` to stick with the naming convention from other modules?
```suggestion
    .collection("prizeGroups")
```